tgui/packages/tgui/interfaces/ChemSeparator.tsx

```diff
@@ -1,159 +1,196 @@
-import { BooleanLike } from 'common/react';
-
 import { useBackend } from '../backend';
-import {
-  Box,
-  Button,
-  LabeledList,
-  NoticeBox,
-  ProgressBar,
-  Section,
-} from '../components';
+import { Box, Button, LabeledList, ProgressBar, Stack } from '../components';
 import { Window } from '../layouts';
 
+type RegHolderData = {
+  total_volume: number;
+  maximum_volume: number;
+  temp: number;
+  color: string;
+};
+
 type Data = {
-  is_burning: BooleanLike;
-  temperature: number;
-  own_total_volume: number;
-  own_maximum_volume: number;
-  own_reagent_color: string;
-  beaker: BooleanLike;
-  beaker_total_volume: number;
-  beaker_maximum_volume: number;
-  beaker_reagent_color: string;
+  flask: RegHolderData;
+  beaker: RegHolderData;
+  fuel: RegHolderData;
+  knob: number;
 };
 
+const BURNER_SETTINGS = [1, 2, 3, 4, 5];
+
 export const ChemSeparator = (props) => {
   const { act, data } = useBackend<Data>();
+  const { flask, beaker, fuel, knob } = data;
+
   return (
-    <Window width={470} height={130}>
+    <Window width={370} height={170}>
       <Window.Content>
-        <Section fill>
-          <LabeledList>
+        <LabeledList>
+          <LabeledList.Item
+            label={
+              <Box
+                style={{
+                  transform: 'translate(20%, -50%)',
+                  width: '57px',
```

Comment:
You have a lot of pixel measurements going on in this UI, initially not a fan. How does it look when resized? 

Is there a way to write this that doesn't require the spacing shimmed with exact measurements (maybe a table)?

Reply:
One way would be to use stack inside a stack, (Like the row would be a stack[label would be a stack item and then the 2nd stack item would be the progress bar/button] and each column would be a stack item and then we could adjust the margins to give us spacing).

But yeah it's just not intuitive as you can see from my explanation